workflow-bot-app/src/test/java/com/symphony/bdk/workflow/custom/assertion/WorkflowAssert.java

```diff
@@ -205,12 +205,15 @@ private void assertOutputs(String key, Object value) {
     String process = lastProcess().orElseThrow();
     await().atMost(5, SECONDS).until(() -> processIsCompleted(process));
 
-    final List<HistoricDetail> historicalDetails =
-        IntegrationTest.historyService.createHistoricDetailQuery().processInstanceId(process).list();
-
-    Optional<HistoricDetail> historicalDetailOptional = historicalDetails.stream()
-        .filter(x -> ((HistoricDetailVariableInstanceUpdateEntity) x).getVariableName().equals(key))
-        .findFirst();
+    Optional<HistoricDetail> historicalDetailOptional = await().atMost(5, SECONDS).until(()
```

Comment:
Do we need a similar fix in https://github.com/finos/symphony-wdk/pull/76/files#diff- 47eda30bc6637c1e8de6cf0b8f4546dbc96c706f202559e36c897e789cf3181dL155 ?

Reply:
It looks like so far we did not have issues with this one, let's keep it in mind if we have flaky tests again